MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofillSuggestion.kt

```diff
@@ -23,13 +24,16 @@ public class AddressAutofillSuggestion internal constructor(
     /**
      * Address geographic point.
      */
-    public val coordinate: Point,
+    public val coordinate: Point?,
 
     /**
      * @suppress
      */
     @JvmSynthetic
     internal val address: AddressComponents,
+
+    @JvmSynthetic
+    internal val underlying: BaseSearchSuggestion?
```

Comment:
Should be non-null, as we always have base suggestion.

Reply:
Done

Reply:
@eugenes78 it's still nullable, please double-check that

Reply:
It turned out to be much trickier than I anticipated. I'm not convinced that I implemented it in the best way possible, so I opened a separate PR just for this comment: #184. Please let me know if you want to bring this change to this branch.